packages/EasyTest/src/Coverage/Factory/CoverageResolverFactory.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyTest\Coverage\Factory;
```

Comment:
Pls make it plural to be consistent with other namespaces.

Reply:
It's Locators now

Reply:
And what about the DataTransferObject namespace?

Reply:
done